scrapy/exporters.py

```diff
@@ -238,9 +244,12 @@ def export_item(self, item):
             self._headers_not_written = False
             self._write_headers_and_set_fields_to_export(item)
 
-        fields = self._get_serialized_fields(item, default_value='',
+        serialized_fields = self._get_serialized_fields(item, default_value='',
                                              include_empty=True)
-        values = list(self._build_row(x for _, x in fields))
+        fields = self._get_fields_from_item(item)
+        if (self.fields_to_export == None or set(fields) != set(self.fields_to_export)) and not self._data_loss_already_warned:
```

Comment:
If `self.fields_to_export` is defined, as per @kmike’s comment [here](https://github.com/scrapy/scrapy/issues/4002#issuecomment-536655863), and INFO statement should be logged instead of a WARNING, because the use has manually chosen a subset of all possible files.

Reply:
Please take a look at this @amardeep24

Reply:
So I am adding an INFO level logger if the user explicitly sets the exported items as a subset. This INFO logger will be issued only once.

```
 if not self._data_loss_already_warned:
            if self.fields_to_export is None:
                logger.warning("Data loss detected when exporting items -- This message won't be shown in further items")
            elif set(fields) != set(self.fields_to_export):
                logger.info("Data loss detected when exporting items -- This message won't be shown in further items")
```